odml/tools/xmlparser.py

```diff
@@ -310,7 +310,12 @@ def parse_tag(self, root, fmt, insert_children=True):
         self.check_mandatory_arguments(check_args, fmt, root.tag, root)
 
         # Instantiate the current odML object with the parsed attributes.
-        obj = fmt.create(**arguments)
+        # Return None, if
+        obj = None
```

Comment:
With setting `obj = None`, currently the parser will probably fail at line 322, if
- there had been an exception in the `obj = fmt.create` step and
- `insert_children` is `True`.